### ISSUE TITLE
Check PDFs and other documents for accessibility

### ISSUE THREAD

**Issue by GeorgiaHCA** [alan-turing-institute/AutSPACEs#301]
### Summary

You’ll need to check your sample of PDFs and other documents (for example, Word documents, spreadsheets and presentations). Your sample should include:

documents that are essential to users accessing one of your services, no matter when they were published

non-essential documents (such as leaflets) that were published on or after 23 September 2018

### Deadline 

20/02/2021

### What needs to be done?

- Check the documents have meaningful titles. First, check the documents have descriptive titles that explain what they’re for.
An example of a good title is something like “Blue badge application form”, as it makes clear what the document is and what a user would use it for. Something like “Accounts - ‘18” is not as good, because it’s vague and does not explain in enough detail what the document is. 
- Check for headings. For this test, you’ll need to open a few of the documents in a document editor. You could use something like Google Docs or Microsoft Word for this if you do not have PDF editing software. The first thing to check is that the document is broken up into sections - and that those sections all have descriptive headings. This will allow people using screen readers to scan the document and jump to the section that’s relevant to them. You’ll also need to check that the headings are tagged properly - for instance, they’ve been created using the styles gallery in Microsoft Word or something similar. That way, a screen reader will recognise them as headings and will let users scan through them to find the content they need. If the headings are just styled using bold, the screen reader will not know they’re headings. 
- Check the documents convey instructions in an accessible way
You need to make sure you’re not conveying instructions in a way that relies on a user’s ability to see the document.
For example, only sighted users will understand instructions like: ‘click the round button’ ‘click the big button below’ ‘click the red button’ Users who cannot see the page will not know what you’re referring to, because you need to be able to see the page to identify a button as ‘big’, ‘red’ or ‘round’. So check your document sample and look out for these types of instructions.
- Check that any link text makes sense. Next, check that any link text you’re using clearly explains where the link will take the user. This is important because screen reader users often scan through lists of links in isolation. This means they do not have the surrounding context to help them understand what the link is for. You can check your links by looking at your sample pages and just reading the link text - without paying attention to the surrounding content. If the links still make sense even in isolation and clearly explain where the links go, it’s likely the text you’re using is accessible. If you’re using link text like ‘click here’ or ‘more information’ then you’re probably not meeting this requirement, as link text like that does not describe where the link will go or what it’s for.
- Check that images, charts and tables have a description .You’ll need to check that any visual way of conveying data like a table, chart or image is described in the surrounding text. That way, users who cannot see the images can still access the information the image conveys.

### Who can help?

Anyone with knowledge and/or time! User testing by users with diverse needs would also be helpful

**Comment by gedankenstuecke** [comment]
Made issue #369 to create an accessibility checklist that can be used for the MVP and going forward. Closing this along with other accessibility checklist items for now.
